Monika After Story/game/event-handler.rpy

```diff
@@ -2240,8 +2657,8 @@ init python:
 
         RETURNS: True if in event list, False if not
         """
-        for ev_data in persistent.event_list:
-            if ev_data[0] == event_label:
+        for item in MASEventList.iter():
+            if item.evl == event_label:
                 return True
 
         return False
```

Comment:
Can't this be replaced with `return mas_findEVL(event_label) != -1`? So we don't repeat our code and will have to maintain later.